include/cudaq/Support/Placement.h

```diff
@@ -70,6 +70,17 @@ class Placement {
     std::swap(phyToVr[phy0.index], phyToVr[phy1.index]);
   }
 
+  void dump(llvm::raw_ostream &os = llvm::errs()) const {
```

Comment:
I think technically we're supposed to flag these with LLVM_DUMP_METHOD or something. We haven't done that anywhere, so just file it away for a future PR. :)